pallets/automation-time/src/fees.rs

```diff
@@ -59,7 +59,6 @@ where
 		prereq: F,
 	) -> Result<R, DispatchError> {
 		let fee_handler = Self::new(owner, action, executions)?;
-		// Note: will need to account for fees in non-native tokens once we start accepting them
```

Comment:
Most of the work for this was done in this pr (https://github.com/OAK-Foundation/OAK-blockchain/pull/325) but the comment was forgotten.